### PR TITLE
Use Equivalent Public ECR Images

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 

Uses Official Public ECR Image for `nginx:latest` .

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ Resources:
           Links:
             - aws-collector
         - Name: traffic-generator-xray
-          Image: 'curlimages/curl:latest'
+          Image: 'public.ecr.aws/itx-devops/curl:latest'
```
Comment (bryan-aguilar) [inline deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml:134]: Who owns `itx-devops`?

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml (hunk 1 above)
Comment (vasireddy99) [reply deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml:134]: Its a mirror image from the Inbound Transportation Experience team, but didn't have full context over ownership.

Comment (bryan-aguilar) [reply deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml:134]: I do not want to introduce a dependency to this ECR repo. If we require the `curl` image I want to depend on officially support images and tags.

Comment (vasireddy99) [reply deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml:134]: The is no available officially supported tags in the Public ECR. I Think this image should be fine as its well used with 50M+ downloads, if not only then only way to use it by mirroring the official image 
image

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**